pkg/storage/storage.go

```diff
@@ -37,6 +39,8 @@ type Storage interface {
 	SaveCertificate(key types.KeyIdentifier, certificate []byte) error
 	GetCertificate(key types.KeyIdentifier) (*x509.Certificate, error)
 	CertificateExists(key types.KeyIdentifier) bool
+	// GetExpiringCertificates lists all certificates will expire between given times
```

Comment:
- that will/are expiring
- document that `till` must be > `from` (otherwise error)